cms/static/js/models/group_configuration.js

```diff
@@ -120,6 +123,10 @@ function(Backbone, _, str, gettext, GroupModel, GroupCollection) {
                     };
                 }
             }
+        },
+
+        groupRemoved: function () {
+            this.setOriginalAttributes();
```

Comment:
@cahrens No the attributes won't be set back to the default case. the ```this.setOriginalAttributes(); ``` function is just setting the current state of attributes of group configuration and related groups.

So if you add two groups into the group configuration it will contains two objects in the related groups collection. The issues was occurring because when you delete one of them the object is deleted from the server but the group collection count stays at two. So after deleting the group if you call  ```setOriginalAttributes() {  this._originalAttributes = this.parse(this.toJSON()); })``` it will set the group count to one. Because ```this.toJSON()``` returns the current state of the object at any point and not the default state

Also why mark ```group_configuration``` model dirty when the  ```group``` object is already deleted on the server?